libs/auth/scopes/src/lib/api.scope.ts

```diff
@@ -4,5 +4,6 @@ export enum ApiScope {
   financeOverview = '@island.is/finance:overview',
   financeSalary = '@island.is/finance/salary',
   internal = '@island.is/internal',
+  representativeWrite = '@island.is/representative:write',
```

Comment:
wondering if the scope should rather be `@island.is/auth/representative:write` ?